custom_components/hubitat/hubitatmaker/hub.py

```diff
@@ -510,6 +515,19 @@ def _get_event_port(port: int | None, event_url: str | None) -> int | None:
         return u.port
     return None
 
+def _get_event_address(address: str | None, host: str | None) -> str | None:
+    """Given an optional address and Hubitat host, return the event address.
+    If no address is provided, open a socket to the Hubitat hub and see what
+    address is used. This would assume this machine and the Hubitat hub are
+    on the same network.
+    """
+    if address is not None:
+        return address
+    if host is not None:
```

Comment:
`host` should never be None since this is only called from the constructor, and the constructor must be passed a host value.